test/1_stdlib/StringDiagnostics.swift

```diff
@@ -98,3 +98,9 @@ func testStringCollectionTypes(s: String) {
   acceptsBidirectionalCollection(s.characters)
   acceptsRandomAccessCollection(s.characters) // expected-error{{argument type 'String.CharacterView' does not conform to expected type 'RandomAccessCollection'}}
 }
+
+struct NotLosslessStringConvertible {}
+
+func testStringInitT() {
+  _ = String(NotLosslessStringConvertible()) // expected-error{{'init' has been renamed to 'init(describing:)}}
```

Comment:
It should provide the fixit hint, but it's actually not providing it right now. 🤔
@CodaFi and I are investigating.